airbyte-webapp/src/components/hooks/services/useDestinationHook.tsx

```diff
@@ -36,7 +36,7 @@ export const useDestinationDefinitionSpecificationLoad = (
       ? {
           destinationDefinitionId
         }
-      : undefined
```

Comment:
Probably `null` is indeed better to be used for such cases and as I remember that's how it is advised to use it in docs. Just a matter of interest - will `undefined` here still work fine or we should always use `null` for such cases?

Reply:
`null` is required because it's explicit. `undefined` might be a mistake like you forgot to pass a second parameter. This helps protect users who don't have typescript enforcing things - as it will fail without the second arg